apps/src/p5lab/AnimationPicker/animation-picker-body.module.scss

```diff
@@ -0,0 +1,28 @@
+@import 'color.scss';
+
+.allAnimations {
+    color: $purple;
```

Comment:
nit: the indentation from this line on is one tab too far indented

Reply:
These kinds of things (indentation and newline) can be caught if we disable prettier ... or re-add these rules in. Do you have a thought about best practice here?

Reply:
ahh, i should've checked the stylelint.config.js file -- now i see "rule-empty-line-before" and "declaration-empty-line-before" are disabled. do you know which rule enforces indentation style?

Reply:
I think it's here https://github.com/stylelint/stylelint-config-standard/blob/main/index.js#L96 but I'm now unable to make any warnings appear ...

Reply:
Oh, I was running the command with --fix so the warnings weren't showing up 🤣 . But yeah, if I disable prettier and then run `npx stylelint "**/*.scss`, then I get warnings like

```
src/lib/ui/Pagination.scss
  3:5   ✖  Expected indentation of 2 spaces  indentation                      
  4:5   ✖  Expected indentation of 2 spaces  indentation                      
  5:5   ✖  Expected indentation of 2 spaces  indentation                      
  7:5   ✖  Expected indentation of 2 spaces  indentation   
```

Reply:
This originally came up with Ben in a PR I did recently https://github.com/code-dot-org/code-dot-org/pull/47716#discussion_r950366826

Reply:
weeeeeeeeird, why does prettier disable that rule? for now, let's just manually fix this and i'll do a pass with the linting rules and make a PR to fix all the existing violations

Reply:
or maybe we should just remove the prettier plugin... the rules it's disabling/conflicting with are ones i feel like we want to keep 🙃

Reply:
I started this PR and can't remember where I left off ... but its goal is to remove prettier. It was fairly straightforward I believe https://github.com/code-dot-org/code-dot-org/tree/remove-prettier